src/test/java/org/painandsuffer/items/armour/chest/ChainmailTest.java

```diff
@@ -0,0 +1,34 @@
+package org.painandsuffer.items.armour.chest;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ChainmailTest {
+    @Test
```

Comment:
Applicable to all Armour test classes.
Since the armour classes share a common structure or behavior, consider using parameterized tests or a shared test template to reduce code redundancy.